src/core/tests/type_prop/squeeze.cpp

```diff
@@ -17,178 +17,230 @@ using namespace std;
 using namespace ov;
 using namespace testing;
 
-TEST(type_prop, squeeze_axes_invalid_value) {
+namespace {
+
+template <typename T>
+class type_prop_local : public testing::Test {};
+
+using Squeeze_v0 = op::v0::Squeeze;
+using Squeeze_v15 = op::v15::Squeeze;
```

Comment:
It can be removed is better to get  tested op by version.

Reply:
Done